plugins/scripts/inject-config.js

```diff
@@ -15,105 +15,110 @@ function createJsonWithDefaultValues(optimoveCredentials, optimoveMobileCredenti
             ? optimoveMobileCredentials
             : "",
         IN_APP_STRATEGY: inAppConsentStrategy,
+         ENABLE_DEFERRED_DEEP_LINKING:
+      !isEmpty(enableDeferredDeepLinking) && isString(enableDeferredDeepLinking)
+        ? enableDeferredDeepLinking
+        : ""
```

Comment:
can use false here instead of "" ? In OptimoveInitProvider we then do `Boolean.parseBoolean(enableDeferredDeepLinking)`

Reply:
yep you are right, done